### ISSUE TITLE
Role invalid when `fifo` is false

### ISSUE THREAD

**Issue by vincentsarago** [mapbox/ecs-watchbot#296]
I believe that in 
https://github.com/mapbox/ecs-watchbot/blob/9e26438d7dc4ed7b2f22f2afcb56d290da51c693/lib/template.js#L377-L382

we happend the policy block in the wrong place. I think it should be: 

```
  if (!options.fifo)
    Resources[prefixed('Role')].Properties.Policies[0].PolicyDocument.Statement.push({
      Effect: 'Allow',
      Action: 'sns:Publish',
      Resource: cf.ref(prefixed('Topic'))
    });
```

cc @rclark

**Comment by rclark** [comment]
Thanks for the report. I'll look at pushing a fix.

**Comment by rclark** [comment]
@vincentsarago the fix was published in v4.13.2

**Comment by vincentsarago** [comment]
thanks @rclark
